Tasks/frmCleanup.cs

```diff
@@ -382,10 +379,69 @@ private void btnCleanup_Click(object sender, EventArgs e)
                 }
             }
 
+            //DNS & ARP
+
+            //FILES
+
             if (checkBox21.Checked)
             {
                 CleanRecentFiles.CleanRecents.ClearAll();
                 CleanupLogsLBox.Items.Add("Recent Files Cleared.");
+            }
+
+            if (checkBox17.Checked)
+            {
+                try
+                {
+
+                    var imgs = Environment.GetFolderPath(Environment.SpecialFolder.MyPictures);
+
+                    foreach (string file in Directory.EnumerateFiles(imgs))
+                    {
+                        try
+                        {
+
+                            File.Delete(file);
+                            CleanupLogsLBox.Items.Add(file + " Deleted!");
+
+                        }
+                        catch(Exception esc)
+                        {
+
+                            CleanupLogsLBox.Items.Add("Failed to delete file" + file + " " + esc);
+
+                        }
+
+                    }
+                    foreach (string file in Directory.EnumerateDirectories(imgs))
+                    {
+                        try
+                        {
+
+                            System.IO.DirectoryInfo dir = new System.IO.DirectoryInfo(file);
+
+                            dir.Delete(true);
+                            CleanupLogsLBox.Items.Add(file + " Deleted!");
+
+                        }
+                        catch (Exception esc)
+                        {
+
+                            CleanupLogsLBox.Items.Add("Failed to delete directory" + file + " " + esc);
+
+                        }
+
+                    }
+                    CleanupLogsLBox.Items.Add("Cleared images folder successfully.");
+
+                }
+                catch(Exception esc)
+                {
+                    CleanupLogsLBox.Items.Add("Failed to clear images folder " + esc);
```

Comment:
I feel like if someone accidentally cleared their images folder, it would probably be on accident. Maybe try adding a feature like the Browser Extention feature.